arm/Microsoft.Network/privateEndpoints/deploy.bicep

```diff
@@ -11,7 +11,7 @@ param serviceResourceId string
 param groupId array
 
 @description('Optional. Resource id of the private DNS zone.')
```

Comment:
Minor: should we update the param description to reflect the change?

Reply:
Fixed.